pkg/cluster/config/default.go

```diff
@@ -58,6 +58,9 @@ func SetDefaults_Cluster(obj *Cluster) {
 		if obj.Networking.IPFamily == "ipv6" {
 			obj.Networking.PodSubnet = "fd00:10:244::/64"
 		}
+		if obj.Networking.IPFamily == "ipv4-ipv6" {
+			obj.Networking.PodSubnet = "10.244.0.0/16,fd00:10:244::/64"
```

Comment:
`fd00:10:244::/64` do you want to make this a /16 because currently the controller manager will default to /24 per node even for IPv6. This is a known problem.